### PR TITLE
Insub | Admin could see a list of Tests, and see their content.

### PR DESCRIPTION
**Pull request by cuongnguyen71190**

### CODE REVIEW

File: app/Http/Controllers/AdminAuth/TestsController.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<?php
```
Comment (tucq88) [inline app/Http/Controllers/AdminAuth/TestsController.php]: Admin's Controllers should not placed inside AdminAuth, it should placed Admin folder instead,

File: app/Models/Test.php [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Test extends Model
+{
+    protected $table = 'tests';
+    protected $fillable = [
```
Comment (tucq88) [inline app/Models/Test.php]: Should not use `fillable`. It's easy to fill out data but also hard to debug.

File: resources/views/admin/test/lists.blade.php [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+                        </thead>
+                        @if (!empty($datas))
+                        <tbody>
+                        <?php $stt=0; ?>
+                        @foreach($datas as $item)
+                        <?php $stt += 1; ?>
+                            <tr class="odd gradeX" align="center">
+                                <td>{!! $stt !!}</td>
+                                <td>{!! $item['title'] !!}</td>
+                                <td>{!! $item['content'] !!}</td>
```
Comment (tucq88) [inline resources/views/admin/test/lists.blade.php]: Should not display content in a table row, if it's too long, the table will look very weird. Instead, try a modal with ajax

Comment (cuongnguyen71190) [inline resources/views/admin/test/lists.blade.php]: at here. can i just display title, status and delete row content ? when click to view button will display full detail.

**Comment by tucq88** [comment]
@cuongnguyen71190 
- No seed for Tests => I see nothing
- No Tests menu tab => I have to manual enter URL

File: app/Http/Controllers/Admin/TestsController.php [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    {
+        $response = array();
+        if ($request->ajax()) {
+            $datas = $this->test->findOrFail($request->id);
+            $response['status'] = 'success';
+            $response['data'] = $datas;
+            return Response::json($response);
+        }
+
+        $datas = $this->test->paginate(2);
```
Comment (tucq88) [inline app/Http/Controllers/Admin/TestsController.php]: By default, each Laravel model will have `$perPage` property as pagination number pre-defined as 50. You could look at root of Model class. 
=> You don't need `2` here

- For typo, `$data` is enough, `$datas` is wrong

File: app/Http/Controllers/Admin/TestsController.php [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index(Request $request)
+    {
+        $response = array();
+        if ($request->ajax()) {
+            $datas = $this->test->findOrFail($request->id);
```
Comment (tucq88) [inline app/Http/Controllers/Admin/TestsController.php]: This Ajax check should sit in show method. Think about it, when you need to look into a record, you should go to show page or list page ?

**Comment by cuongnguyen71190** [comment]
i want change some things about file and folder structure, so that, i close this to create new pull request

**Pull request closed without merging**